database/migrations/platform/2023_11_20_000000_update_player_stats_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('player_stats', function (Blueprint $table) {
+            $table->timestampTz('last_affected_at')->nullable()->after('last_game_id');
+        });
+    }
```

Comment:
I'm not sure if we should do this; it doesn't seem right to have a DB write operation occur in a migration. None of our other migrations contain DB writes, so I feel like this might be precedent-setting.

I recommend on deploy we simply run the full sync command again for `player_stats`. If I recall correctly, when it was last run, it finished in only a few minutes.

Reply:
database migrations shouldn't include data migrations, only schema changes.

data migrations should be done in re-usable commands or as one-off queries right after deployment.

running the command after deployment sounds good to me.

Reply:
personally, i'd go with `stats_updated_at` to specify a timestamp bound to business logic. as mentioned, created_at and updated_at are framework/system timstamps.

when introducing update timestamps for a subset of columns or something like in this case i tend to introduce one or multiple `x_updated_at` timestamps to keep naming consistent.

Reply:
Column name change addressed in https://github.com/RetroAchievements/RAWeb/pull/2050/commits/36175efd856d991bfe0955d83a4440f8233aa285.

Note: I opted for `stat_updated_at` (singular), because each row represents a single statistical data point.

I do believe we should run the command on deploy to restore the data integrity.